docs-source/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -149,6 +149,7 @@ The following parameters can be provided in the inputs file.
 | `t3ChannelPort` | Port for the T3 channel of the NetworkAccessPoint. | `30012` |
 | `t3PublicAddress` | Public address for the T3 channel.  This should be set to the public address of the Kubernetes cluster.  This would normally be a load balancer address. <p/>For development environments only: In a single server (all-in-one) Kubernetes deployment, this may be set to the address of the master, or at the very least, it must be set to the address of one of the worker nodes. |  If not provided, the script will attempt to set it to the IP address of the kubernetes cluster |
 | `weblogicCredentialsSecretName` | Name of the Kubernetes secret for the Administration Server's user name and password. | `domain1-weblogic-credentials` |
+| `serverPodCpuRequest`, `serverPodMemoryRequest`, `serverPodCpuCLimit`, `serverPodMemoryLimit` |  The maximum amount of compute resources allowed, and minimum amount of compute resources required for each server pod. Please refer to kubernetes documentation on `Managing Compute Resources for Containers` for details. | Resource requests and resource limits are not specified |
```

Comment:
required for each -> required, for each
refer to kubernetes documentation -> refer to the Kubernetes documentation
specified -> specified. (period)